lib/views/home/widgets/bottom_sheet/stats/stats_bottom_sheet_widget.dart

```diff
@@ -46,52 +41,48 @@ class StatsBottomSheetWidget extends ConsumerWidget {
               ),
             ),
           ],
-        );
-      },
+        ),
+      ),
     );
   }
 
-  Expanded _statsList(
+  Column _statsList(
     BuildContext context,
-    ScrollController scrollController,
     GlobalKey key,
     StatsModel stats,
   ) {
-    return Expanded(
-      child: SingleChildScrollView(
-        controller: scrollController,
-        child: Column(
-          children: [
-            RepaintBoundary(
-              key: key,
-              child: Container(
-                color: ColorConstants.onyx,
-                child: Column(
-                  children: stats.all.map((e) {
-                    return RowItemWidget(
-                      iconCodePoint: e.icon,
-                      iconColor: e.color,
-                      iconSize: 20,
-                      title: e.title,
-                      subTitle: e.subtitle,
-                      hasUnderline: e.title != stats.all.last.title,
-                      isTrailingIcon: false,
-                      titleStyle: Theme.of(context)
-                          .textTheme
-                          .labelMedium
-                          ?.copyWith(fontSize: 18),
-                    );
-                  }).toList(),
-                ),
-              ),
-            ),
-            ShareBtnWidget(
-              globalKey: key,
-              shareText: StringConstants.shareStatsText,
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        RepaintBoundary(
+          key: key,
+          child: Container(
+            color: ColorConstants.onyx,
+            child: Column(
+              children: stats.all.map((e) {
+                return RowItemWidget(
+                  iconCodePoint: e.icon,
+                  iconColor: e.color,
+                  trailingIconSize: 20,
```

Comment:
should this be in a constant?

Reply:
Created a constant for trailingIconSize.